tools/unsafe_optimizations.js

```diff
@@ -299,7 +299,7 @@ function runTests() {
   // Interaction between multiple passes:
   test(
     'var d, f; f = new Uint8Array(16); var h = f.buffer; d = new Uint8Array(h);',
-    'var f=new Uint8Array(16),h=f.buffer,d=new Uint8Array(h);'
+    'var f=new Uint8Array(16),h=f.buffer,d=new Uint8Array(h);',
```

Comment:
This is just the result of the latest eslint/prettier npm modules running out our code.

Reply:
Without this the linter will now fail.

Reply:
Huh... but isn't it uglier now? It's doing this:
```js
test(
  foo,
  bar,    // weird last comma here
);
```
Is it just me that dislikes that?

Reply:
I did at first too. It grew on me eventually, since it makes diffs smaller when adding a new item. It also makes it possible to swap lines up and down.

Reply:
Ah, good point about diffs. So this doesn't do `test(foo, bar,)` on one line, but only when split... lgtm.

Reply:
> I did at first too. It grew on me eventually, since it makes diffs smaller when adding a new item. It also makes it possible to swap lines up and down.

Yes I hugely perfer this in both JS and Python for this reason.  Make all the lines in the list the same and makes adding lines to end a less intrusive diff.

If only json supported this!